### PR TITLE
Inidivudal Sound Volumes

### PR DESCRIPTION
**Pull request by qwoodmansee**
_**DO NOT MERGE**_ until the initial value of slider is fixed.

Absolute MVP of individual sound levels. The only thing that **functionally isnt working** is the initial values of the sliders. I've never worked in handlebars before, only spent an hour on this, and figured the original addon devs would know what I'm doing wrong rather quickly. I'll add an inline comment where the problem is.

Here's a screenshot:
Screen Shot 2021-05-17 at 11 08 14 PM

### CODE REVIEW

File: templates/soundboard.html [hunk 1]
```diff
@@ -141,6 +141,8 @@ <h3><small class="text-muted" style="
             onclick="SoundBoardApplication.toggleExtendedOptions(this, '{{identifyingPath}}', '{{../../tab.fav}}')">
             <i class="fas fa-ellipsis-v"></i>
           </button>
+          <input type="range" min="1" max="100" value="SoundBoard.getVolumeForSound('{{identifyingPath}}')" class="slider col" id="volumeRange-{{identifyingPath}}"
```
Comment (qwoodmansee) [inline templates/soundboard.html:144]: this `value=` isn't working, so we are getting the default value of 50% which is annoying

File: soundboard.js [hunk 2]
```diff
@@ -138,7 +152,7 @@ class SoundBoard {
     static async playSound(identifyingPath, push = true) {
 
         let sound = SoundBoard.getSoundFromIdentifyingPath(identifyingPath);
-        let volume = SoundBoard.getVolume();
+        let volume = SoundBoard.getVolumeForSound(identifyingPath) ?? SoundBoard.getVolume();
```
Comment (qwoodmansee) [inline soundboard.js:155]: i realize in reality this logic should _probably_ be a combination of the `volumeForSound` _and_ the global volume, so turning down the global volume still effects sounds with individual volumes.

Figured I'd let you guys decide how exactly you wanted that to work

Comment (qwoodmansee) [reply soundboard.js:155]: I also just realized that I implemented the fallback both here and `getVolumeForSound`. That's why we have draft PRs haha

**Comment by BlitzKraig** [comment]
Hi qwoodmansee,

Thanks for the contribution. As discussed on Discord, I want to make a couple of changes to this, mainly hiding it behind a toolbar option.

I'm going to merge into the 1.5.1-rc branch and make the changes for the 1.5.1 release. Additionally, I'll take a look at the bug you've mentioned in the comments.

Thanks again

**Pull request merged**